src/background/portManager.ts

```diff
@@ -172,6 +173,31 @@ export async function getProcessNameFromPid(
   return processName.trim();
 }
 
+/**
+ * ポートが割り当て可能かどうか実際にlistenして確認します。
+ * @param hostInfo ホスト情報
+ * @returns 割り当て不能だった場合`undefined`を返します。割り当て可能だった場合ポート番号を返します。
+ */
+function testPort(hostInfo: HostInfo): Promise<number | undefined> {
+  return new Promise((resolve, reject) => {
+    const server = createServer();
+    server.on("error", () => {
+      server.close();
+      resolve(undefined);
+    });
+    server.on("listening", () => {
+      const address = server.address();
+      server.close();
+      if (address == undefined || typeof address === "string") {
+        reject(new Error("'address' is null or string"));
```

Comment:
これちょっと念のためにエラーメッセージをローリングできるようにしとくのどうでしょう。なんか変な文字列が来るかもしれないので。。
```suggestion
        reject(new Error(`'address' is null or string: ${address}`));
```